test/rules/_integration/enable-disable/test.ts.lint

```diff
@@ -78,9 +78,6 @@ var AAAaA = 'test' // previously both `quotemark` and `variable-name` rules were
 var AAAaA = 'test' // tslint:disable-line:quotemark
 var AAAaA = 'test' // ensure that disable-line rule correctly handles previous `enable rule - disable all` switches
 
-// tslint:enable:no-var-keyword
```

Comment:
The behaviour to not load disabled rules was added recently. Even though the rule was loaded, it would have never been executed, because `rule.isEnabled()` always returned false. I don't know why the test was implemented this way...
I think it makes more sense to keep disabled rules disabled. Otherwise you need to guess the `ruleSeverity`